cpuid/src/cpu_leaf.rs

```diff
@@ -43,18 +52,18 @@ pub mod leaf_0x1 {
         pub const PDCM_SHIFT: u32 = 15;
         // 18 = DCA Direct Cache Access (prefetch data from a memory mapped device)
         pub const MOVBE_SHIFT: u32 = 22;
-        pub const TSC_DEADLINE_TIMER_SHIFT: u32 = 24;
+        pub const TSC_DEADLINE_TIMER: u32 = 24;
         pub const OSXSAVE_SHIFT: u32 = 27;
         // Cpu is running on a hypervisor.
-        pub const HYPERVISOR_SHIFT: u32 = 31;
+        pub const HYPERVISOR: u32 = 31;
```

Comment:
Also, it becomes confusing to know which are just indexes and which are ranges. `HYPERVISOR` here is an index, but `APICID` for example is a range.

What do you think about adding a name suffix for easier identification?